Framework/API/inc/MantidAPI/DistributedAlgorithm.h

```diff
@@ -4,6 +4,7 @@
 //   NScD Oak Ridge National Laboratory, European Spallation Source,
 //   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 // SPDX - License - Identifier: GPL - 3.0 +
+#define BOOST_BIND_GLOBAL_PLACEHOLDERS
```

Comment:
Can I ask why we are defining this in a header globally? Can we not switch completely away from `boost::bind` now?

Reply:
We needed to define this globally as a boost header is using boost::bind in a way that isn't supported by default without this being added.

Reply:
I am going to attempt to define this globally using CMake like you suggest for BOOST_ALLOW_DEPRECATED_HEADERS